components/suggestion/suggestion_list.jsx

```diff
@@ -84,7 +84,7 @@ export default class SuggestionList extends React.PureComponent {
         this.maxHeight = Math.min(
             window.innerHeight - (inputHeight + Constants.POST_MODAL_PADDING),
             Constants.SUGGESTION_LIST_MAXHEIGHT,
-        );
+        ) + 300;
```

Comment:
please don't use arbitrary values.

Reply:
Sorry. I didn't get your point. Since the ticket says increase the max-height by 300 pixels, therefore I just used it. Can you suggest me a proper way of doing this? Thanks.

Reply:
well, a line above your change it literally says `Constants.SUGGESTION_LIST_MAXHEIGHT` ... this is where the value should be adjusted. Arbitrary values are insecure since they only apply to a specific case. If we want to use the Constant value somewhere else we would run into issues since this value and the one from the constant will not match.

Additionally, and I did not catch that earlier, this change will increase the minimum value by `300px` which is not the intention. Only the maximum height should be changed.

So the change that should have been made is in a different file.

Reply:
Thanks for the reply. So basically you mean the changes should be like this
`
this.maxHeight = Math.min(
            window.innerHeight - (inputHeight + Constants.POST_MODAL_PADDING),
            Constants.SUGGESTION_LIST_MAXHEIGHT+Constants.TO_DO
        );
`
Am I correct?

Reply:
@michelengelen got your point. So I need to change
`
SUGGESTION_LIST_MAXHEIGHT: 292
`
to 
`SUGGESTION_LIST_MAXHEIGHT: 592
`

Am I correct?

Reply:
@sharmadhiraj86 sry for answering this late ... still goinf through all the notifications I missed. 

This change would be sufficient, yes.